docs/TargetPlugin.md

```diff
@@ -0,0 +1,27 @@
+## [TargetPlugin](../src/TargetPlugin.groovy)
+
+Enable this plugin to run plan/apply on selective resource targets
+
+```
+// Jenkinsfile
+@Library(['terraform-pipeline']) _
+
+Jenkinsfile.init(this, env)
+TargetPlugin.init() // Optionally limit plan/apply to specific targets
+
+
```

Comment:
Nit picky - are you intended extra whitespace?

If it for clarify, I'm good.  If it's a typo, I'd prefer it be cleaned up.

Reply:
Yeah I just had the extra lines in there for visibility. Mainly to show there is the 'init' part of the code followed by the actual running of the pipeline. I really don't feel too strongly either way. I can clean it up if you prefer.